_data/simple-icons.json

```
@@ -10481,6 +10481,11 @@
             "hex": "E05E00",
             "source": "https://www.newegg.com"
         },
+        {
+            "title": "NEXON",
```

```suggestion
            "title": "Nexon",
```
This seems to be how the company capitalizes its name. Make sure to update title and `<title>` as well.